presto-spi/src/main/java/com/facebook/presto/spi/PrestoException.java

```diff
@@ -40,6 +46,11 @@ public PrestoException(ErrorCodeSupplier errorCodeSupplier, String message, Thro
         this.errorCode = errorCodeSupplier.toErrorCode();
     }
 
+    public SerializedPrestoException getSerialized()
```

Comment:
it is not a getter so I would name it `toSerialized()` or even without shortcut `toSerializedPrestoException()`